spec/system/admin_dashboard_new_features_spec.rb

```diff
@@ -30,6 +30,8 @@
     expect(new_features_page).to have_screenshot
     expect(new_features_page).to have_learn_more_link
     expect(new_features_page).to have_no_emoji
+    expect(new_features_page).to have_date("November 2023")
+    expect(new_features_page).to have_version("3.3.0.beta4")
   end
```

Comment:
Maybe also have a spec that shows things are grouped correctly by month by having releases span over multiple months?